documentation/getting_started/configuring-addresses.adoc

```diff
@@ -18,7 +18,7 @@ echo "https://$({OcGetRoute} console)"
 ....
 
 You can also deploy the addressing config using the REST API
-API. See the <<configuring-using-restapi>> for details on the
+API. See xref:configure-address-spaces-addresses-restapi-messaging[] for details on the
 resources consumed by the API. Here is an example config with all 4 variants that you can save to `addresses.json`:
```

Comment:
For details of the resources consumed by the API, see xref:configure-address-spaces-addresses-restapi-messaging[].

Reply:
Fixed